ui/src/vertical-geometry/labeled-ticks.tsx

```diff
@@ -47,7 +54,90 @@ const TickLabel: React.FC<{
     );
 };
 
-export const LabeledTicks: React.FC<LabeledTicksProps> = ({ trackKmHeights, coordinates }) => {
+const PlanLinkingDividers: React.FC<{
+    planLinkingSummary: PlanLinkingSummaryItem[] | undefined;
+    coordinates: Coordinates;
+    startM: number;
+    endM: number;
+}> = ({ planLinkingSummary, coordinates, startM, endM }) => {
+    if (!planLinkingSummary) {
+        return <React.Fragment />;
+    }
+
+    return (
+        <>
+            {planLinkingSummary.map((summary, i) => {
```

Comment:
Jos tähän laittaa planLinkingSummary?.map.., niin ylempää if-ehtoa ei tarvita.

Reply:
Jeps. Näitä oli muissakin paikoissa jotka muuttelin käyttämään kyssäriä.